File: push/src/main/java/com/xiaomi/clientreport/processor/IPerfProcessor.java
package com.xiaomi.clientreport.processor;

import com.xiaomi.clientreport.data.BaseClientReport;
import java.util.HashMap;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/clientreport/processor/IPerfProcessor.class */
public interface IPerfProcessor extends IDataSend, IWrite {
    void setPerfMap(HashMap<String, HashMap<String, BaseClientReport>> map);
}

File: legacy/src/main/java/com/xiaomi/clientreport/processor/IPerfProcessor.kt
package com.xiaomi.clientreport.processor
import io.github.magisk317.mipush.protocol.model.*

import com.xiaomi.clientreport.data.BaseClientReport
import java.util.HashMap

interface IPerfProcessor : IDataSend, IWrite {
    fun setPerfMap(map: HashMap<String, HashMap<String, BaseClientReport>>)
}
